packages/synapse-react-client/src/components/SynapseHomepageV2/SynapseHomepageV2.tsx

```diff
@@ -0,0 +1,514 @@
+            minHeight: '600px',
+            width: '100%',
+            height: '100%',
+            content: "''",
+            top: 0,
+            right: 0,
+            bottom: 0,
+            left: 0,
+            zIndex: -1,
+            clipPath: 'polygon(0 0, 100% 0, 100% 65%, 0% 100%)',
+          }}
+        >
+          {' '}
+        </Box>
+      </Box>
+
+      <Box
+        sx={{
+          textAlign: 'center',
+          p: {
+            xs: '20px',
+            lg: '80px 0px 0px 0px',
+          },
+          minHeight: {
+            xs: '270px',
+            md: undefined,
+          },
+        }}
+      >
+        <Typography variant="headline1" sx={titleSx}>
+          Explore the data
+        </Typography>
+        <Typography variant="headline1" sx={titleSx}>
+          behind&nbsp;
+          <TypeAnimation
+            sequence={[
+              // Same substring at the start will only be typed out once, initially
+              'the next cure',
+              3000,
+              'the next diagnostic',
+              3000,
+              'the next preventive therapy',
+              3000,
+            ]}
+            wrapper="span"
+            speed={20}
+            repeat={Infinity}
+            style={{
+              color: '#B5D3CE',
+            }}
+          />
+        </Typography>
+      </Box>
+
+      {/* Search */}
+      <Box
+        sx={{
+          pt: '70px',
+          maxWidth: '600px',
+          m: 'auto',
+        }}
+      >
+        <SynapseHomepageSearch sourceTable={searchAutocompleteTable} />
+      </Box>
+      <Box
+        sx={{
+          display: 'flex',
+          gap: '10px',
+          alignItems: 'center',
+          justifyContent: 'center',
+          mt: '30px',
+          flexWrap: 'wrap',
+        }}
+      >
+        {/* Hard-coded popular searches because these Chips are above the fold and immediately visible. Any delay showing these chips is a problem.
+        The hope is that these "popularSearches" will not change much over time, since they represent the types of data in the platform.
+        */}
+        {popularSearches.map(value => {
+          return (
+            <Chip
+              key={value}
+              label={value}
+              onClick={() => onSearch(value)}
+              variant="outlined"
+              // by default, on hover the background color changes to mostly transparent (4%), which looks terrible on top of the header splash image
+              sx={{
+                color: '#2A5850',
+                backgroundColor: '#DAE9E7',
+                '&:hover': { backgroundColor: '#f5f5f3 !important' },
+              }}
+            />
+          )
+        })}
+      </Box>
+      <Box
+        sx={{
+          display: { xs: 'relative', lg: 'grid' },
+          gridTemplateColumns: '50% 50%',
+          backgroundColor: '#DAE9E7',
+          mt: '150px',
+          height: { lg: '608px' }, //force container to the same height as the image
```

Comment:
is this because the image happens to be 608px? Can we apply this height to both elements in case the image changes?